templates/rosetta_finetuning.ipynb

```diff
@@ -110,14 +110,9 @@
     "panel = pd.read_csv(os.path.join(base_dir, panel_file_name))\n",
```

Comment:
<p>This will fail because of a non-keyword argument, make sure you always run each notebook whenever you make changes to it since we're not using travis to test them</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/130/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>